packages/@okta/vuepress-theme-prose/components/ThemeModeToggle.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <label class="toggle-switch switch-theme">
+    <span :class="{'light-mode': true, 'active': !isDarkMode}" @click="toggleDarkMode(false)"></span>
+    <span :class="{'dark-mode': true, 'active': isDarkMode}" @click="toggleDarkMode(true)"></span>
+  </label>
+</template>
+
+<script>
+import { isSetCookie, getCookie, setCookie } from "../util/attribution/cookies";
```

Comment:
I'm not sure this cookie util is the correct tool for this. I'm not exactly sure how the attribution values are being used on the backend but, the `theme` value could leak over there.

Possible to store this in `local storage`?

Reply:
Sure, will use localStorage